_docs/faq.markdown

```diff
@@ -3,17 +3,17 @@ title: FAQ
 order: 999999
 ---
 
-{% include h title="Where is Rust SDK?" tag="sdk" %}
+{% include h title="Where is the Rust SDK?" tag="sdk" %}
```

Comment:
As a fellow non-native speaker, I understand the feeling, but that's definitely not the right grammar :D.